bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/RuleEnableModuleType.json

```diff
@@ -0,0 +1,26 @@
+{
+	"actions": [
+		{
+			"uid": "RuleEnableAction",
+			"label": "Enables or disables rules",
+			"description": "This handler can enable or disable a rule or group of rules specified by ID.",
+			"configDescriptions": [
+				{
+					"name": "action",
+					"type": "TEXT",
```

Comment:
Wouldn't a boolean be much better here? Additionally, there should be a list of options provided.